specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/datafactory.json

```diff
@@ -375,6 +375,93 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataFactory/factories/{factoryName}/sharedFactory": {
+      "post": {
+        "tags": [
+          "factories"
+        ],
+        "operationId": "Factories_ListSharedFactory",

```

Comment:
This operation should use the x-ms-pageable extension

Reply:
@annatisch Done.